pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -223,6 +233,7 @@ func (c *Cluster) makeMgrSidecarContainer(mgrConfig *mgrConfig) v1.Container {
 			},
 		}},
 		{Name: "ROOK_DASHBOARD_ENABLED", Value: strconv.FormatBool(c.spec.Dashboard.Enabled)},
+		{Name: "ROOK_DASHBOARD_EXTERNAL_IPS", Value: externalIps},
```

Comment:
With the `strings.Join()` we shouldn't need the for loop above
```suggestion
		{Name: "ROOK_DASHBOARD_EXTERNAL_IPS", Value: strings.Join(externalIps, ",")},
```